src/pages/søknad/steg/inntekt/Inntekt.tsx

```diff
@@ -372,6 +373,11 @@ const DinInntekt = (props: { forrigeUrl: string; nesteUrl: string; avbrytUrl: st
                     onSubmit={(e) => {
                         setHasSubmitted(true);
                         formik.handleSubmit(e);
+                        setTimeout(() => {
+                            if (feiloppsummeringref.current) {
+                                feiloppsummeringref.current.focus();
+                            }
+                        }, 0);
```

Comment:
Vad er disse till för? 😊

Reply:
De er for at feiloppsummeringen skal få fokus etter at validering har kjørt. Glemte å oppdatere PR-navnet for å reflektere dette :p